docs/source/markdown/podman-kube-play.1.md.in

```diff
@@ -163,6 +163,10 @@ and as a result environment variable `FOO` is set to `bar` for container `contai
 
 Build images even if they are found in the local storage. Use `--build=false` to completely disable builds. (This option is not available with the remote Podman client)
 
+       Note:  You  can also override the default isolation type by setting the
```

Comment:
Should this be indented? I think the md2man processor strips that.

Reply:
@rhatdan ^ is the indent intended?

Reply:
No